integration-tests/push-to-external-registry-idempotent/test.env

```diff
@@ -0,0 +1,28 @@
+uuid=${uuid:-"$(openssl rand -hex 4)"}
+# make sure uuid is 8 chars long
+uuid="${uuid:0:8}"
+
+export originating_tool="push-to-external-registry-idempotent-e2e-test"
+
+# Namespaces
+export tenant_namespace=dev-release-team-tenant
+export managed_namespace=managed-release-team-tenant
+
+export application_name=push-external-app-${uuid}
```

Comment:
A different application name is needed since all the test cases will be run parallelly in nightly test.

Reply:
absolutely! good catch ! thanks!
fixed